content/en/docs/tasks/administer-cluster/controller-manager-leader-migration.md

```diff
@@ -0,0 +1,109 @@
+---
+reviewers:
+- jpbetz
+- cheftako
+title: "Controller Manager Leader Migration"
+linkTitle: "Controller Manager Leader Migration"
+content_type: task
+---
+
+
+<!-- overview -->
+
+{{< feature-state state="alpha" for_k8s_version="v1.21" >}}
+
+{{< glossary_definition term_id="cloud-controller-manager" length="all" prepend="The cloud-controller-manager is">}}
+
+## Background
+
+As part of the cloud provider extraction effort, all cloud specific controllers must be moved out of the `kube-controller-manager`. All existing clusters that run cloud controllers in the `kube-controller-manager` must migrate to instead run the controllers in a cloud provider specific `cloud-controller-manager`.
+
+Leader Migration provides a mechanism in which HA clusters can safely migrate "cloud specific" controllers between the `kube-controller-manager` and the `cloud-controller-manager` via a shared resource lock between the two components.
+
+This guide walks you through the process of upgrading the control plane from `kube-controller-manager` with built-in cloud provider to running both `kube-controller-manager` and `cloud-controller-manager`.
+
+## {{% heading "prerequisites" %}}
+
+The control plane nodes should run `kube-controller-manager` with Leader Election enabled through `--leader-elect=true`, and the cloud privider set with `--cloud-provider` flag.`cloud-controller-manager` should not yet be deployed.
+
+<!-- steps -->
+
+### Grant Access to Migration Lease
```

Comment:
nit: headings should be sentence case (but _page_ titles are title case).

Reply:
Thank you. Updating.